lib/heroku/helpers/addons/api.rb

```diff
@@ -0,0 +1,98 @@
+module Heroku::Helpers
+  module Addons
+    module API
+      VERSION="3.switzerland".freeze
```

Comment:
This should be changed to just `"3".freeze`, now that the variant has been merged.